src/index.js

```diff
@@ -10,6 +10,9 @@ import UsersView from "./views/UsersView";
 import SettingsView from "./views/SettingsView";
 import PropTypes from 'prop-types';
 
+/* Offline-plugin allows offline availability.
+Only loaded in prod environment (see webpack.prod.js)
+to allow hot reloading of webpack-dev-server */
 const runtime = require('offline-plugin/runtime');
```

Comment:
Fortunately runtime does check for plugin availability for itself, thus we do not need to disable it here

Reply:
Nice, also good to have a comment here!